corehq/warehouse/transforms/sql/app_status_fact.sql

```diff
@@ -0,0 +1,45 @@
+SELECT * FROM
```

Comment:
gotcha, heh that's gonna make for a lot more staging tables. would make any sense to use materialized views for our temporary tables?